packages/protocol/migrationsConfig.js

```diff
@@ -51,7 +52,7 @@ const DefaultConfig = {
     targetVotingYieldParameters: {
       initial: 0.00016, // (x + 1) ^ 365 = 1.06
       max: 0.0005, // (x + 1) ^ 365 = 1.20
-      adjustmentFactor: 1 / 365,
+      adjustmentFactor: 1 / 3650,
```

Comment:
We never really came to a conclusion on what we wanted to do for transitioning from the genesis to elected validator set in RC0. My suggestion would be to lower the `minElectableValidators` for now to a very low number, e.g. 5, with a comment that it should be raised to 50 for mainnet